lib/generators/solidus_friendly_promotions/install/templates/initializer.rb

```diff
@@ -51,5 +51,6 @@
 end
 
 SolidusFriendlyPromotions.configure do |config|
-  # Add your custom configuration here
+  # Sync order promotions between the legacy and new promotion systems
```

Comment:
Maybe add a comment about removing it when migration is done

Reply:
Added